packages/client/src/helpers/client/interceptors/authentication/__tests__/AxiosAuthenticationTokenManager.test.js

```diff
@@ -679,10 +679,9 @@ describe('AxiosAuthenticationTokenManager', () => {
     it('should throw an error if a request is performed before token manager is loaded', () => {
       tokenManagerInstance = createAndSetTokenManagerInstance(client);
 
-      expect(tokenManagerInstance.isLoaded).toBeFalsy();
-
-      expect.assertions(1);
+      expect.assertions(2);
```

Comment:
Question: Why was this necessary?

Reply:
This test failed after msw config. But, if you check, there are two expects in this test. I'm curious how did the test pass before this configuration...

Reply:
After the new config setup for msw, one assertion was only needed but this is strange because you have two expects in the same test.

Reply:
Interesting question. I dig a little deeper in jest to understand why the test was failing now when it was passing before and the thing is, the added `afterEach` call to setupAfterEnv.js will change the timings internally in jest, namely when it will process the assertions. Without the `afterEach` call it will process the results right before the call to expect(getUser()).rejects... which will reset the internal assertions number to 0 while with the `afterEach` call it will only do that after all expects are done.

TLDR; 
The test code was in fact incorrect, it should be `expect.assertions(2)` as you have done to avoid these timings problems in jest depending on the kind of tests being done and the afterEach blocks defined. Closing it... 

Thanks!